### DISCUSSION TITLE
Expedition 2

### DISCUSSION THREAD

**Discussion by PeAfMa-p** [leodium/leodium-ReLive-NoMansSky-Expeditions-Offline#5] [General]
In order to finish the expedition, we need to buy an item with 80 quicksilver, but the nexus doesn't seem to give quicksilver missions when offline. The only reliable way to get quicksilver on the expedition is by doing 3 "flux readings" in an earlier milestone, but each reading only awards 15 quicksilver, which is not enough. It is possible to keep doing readings in order to get enough quicksilver, but some people (like me) may not realize that. If possible, it would be nice to increase the amount of quicksilver rewarded, increase the number of readings required, or put in the readme file that it is possible to keep doing readings.

**Comment by GitFo0** [comment]
Hey bud, you can enable nexus quicksilver missions on your offline save files https://www.gog.com/forum/no_mans_sky/how_to_enable_expeditions_quicksilver_missions_with_offline_gog_installer_in_2024 this can be done in or outside of expeditions fwict. You'd just need to click the folder named `ExpeditionContext` instead of `BaseContext` when following the steps in the section on enabling quicksilver missions in the link i've provided.

**Comment by cwmonkey** [comment]
In the new Redux version, they now give more quicksilver for other milestones.
